src/Makefile.am

```diff
@@ -548,22 +552,22 @@ libbitcoin_cli_a_SOURCES = \
 nodist_libbitcoin_util_a_SOURCES = $(srcdir)/obj/build.h
 #
 
-# bitcoind binary #
-bitcoind_SOURCES = bitcoind.cpp
-bitcoind_CPPFLAGS = $(AM_CPPFLAGS) $(BITCOIN_INCLUDES)
-bitcoind_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
-bitcoind_LDFLAGS = $(RELDFLAGS) $(AM_LDFLAGS) $(LIBTOOL_APP_LDFLAGS)
+# bitcoind, bitcoin-wallet, bitcoin-node binaries #
+bitcoind_common_sources = bitcoind.cpp
```

Comment:
Please make these uppercase so they stand out from automake variables. As-is, this looks like a new helper lib.

Reply:
> Please make these uppercase so they stand out from automake variables. As-is, this looks like a new helper lib.

I don't think this works. If I make everything uppercase I see errors:

```
src/Makefile.am:568: warning: variable 'BITCOIND_COMMON_LDFLAGS' is defined but no program or
src/Makefile.am:568: library has 'BITCOIND_COMMON' as canonical name (possible typo)
```

These are lowercase specifically because automake seems to pick up uppercase suffixes as target properties.

Tested with:

```bash
for f in bitcoind_common_sources bitcoind_common_cppflags bitcoind_common_cxxflags bitcoind_common_ldflags bitcoin_qt_common_cppflags bitcoin_qt_common_cxxflags bitcoin_qt_common_sources bitcoin_qt_common_ldadd bitcoin_qt_common_ldflags bitcoin_qt_common_libtoolflags; do
git grep -l $f | xargs sed -i "s/$f/$(echo $f | tr '[:lower:]' '[:upper:]')/g"
done
```

Reply:
Agree that naming of convenient variables with `all_lower_case` makes them distinguishable from the Automake special variables.